### PR TITLE
Added r4 consent to fhir.cerner.com

### PR DESCRIPTION
**Pull request by shubhamvinayak**
Description
----
Added r4 consent to fhir.cerner.com

Screenshot before merge
-----
image
image
image
image
image

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: lib/resources/example_json/r4_examples_consent.rb [hunk 1]
```diff
@@ -0,0 +1,814 @@
+# frozen_string_literal: true
+
+module Cerner
+  module Resources
+
+    # TODO: Missing data
```
Comment (ashu-bdvt) [inline lib/resources/example_json/r4_examples_consent.rb:6]: Remove TODOs

File: content/millennium/r4/entities/consent.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+<%= headers status: 200 %>
+<%= json(:r4_consent_entry) %>
+
+<%= disclaimer %>
+
+#### Patient Authorization Request
+
+#### Request
+
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Consent/AR-12724069-12724066
```
Comment (aki-mathur) [inline content/millennium/r4/entities/consent.md:174]: Update it to https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Consent/AR-12724069-12724066

**Review by aki-mathur — CHANGES_REQUESTED** [review]

File: lib/resources/example_json/r4_examples_consent.rb (hunk 1 above)
Comment (shubhamvinayak) [reply lib/resources/example_json/r4_examples_consent.rb]: comment addressed

File: content/millennium/r4/entities/consent.md (hunk 2 above)
Comment (shubhamvinayak) [reply content/millennium/r4/entities/consent.md]: https://github.com/cerner/fhir.cerner.com/pull/509/commits/4b0037b680117517803a7dcb189943d35ddb8aaf
comment addressed

**Review by aki-mathur — APPROVED** [review]

**Review by ashu-bdvt — APPROVED** [review]

**Pull request merged**